cmd/fabric-ca-server/config.go

```diff
@@ -578,6 +579,20 @@ func (s *ServerCmd) configInit() (err error) {
 		return err
 	}
 
+	// Info logging for CA config, includes yaml file settings and environment variable overrides
+	allSettings := s.myViper.AllSettings()
+	settingsYaml, err := yaml.Marshal(allSettings)
+	if err == nil {
+		log.Infof("CA config with combined yaml file and environment variable overrides:\n%s", settingsYaml)
+	}
+
+	// Debug logging for CA environment variables
+	log.Debugf("Environment variables:")
+	envVars := os.Environ()
+	for _, envVar := range envVars {
+		log.Debug(envVar)
+	}
```

Comment:
This can also leak passwords in environment variables.

Reply:
ok, I've removed the env variables.